app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayViewModel.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.navigationoverlay
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.Transformations
+import android.arch.lifecycle.ViewModel
+import org.mozilla.tv.firefox.session.SessionRepo
+import org.mozilla.tv.firefox.utils.URLs
+
+class NavigationOverlayViewModel(sessionRepo: SessionRepo) : ViewModel() {
+    val viewIsSplit: LiveData<Boolean> = Transformations.map(sessionRepo.state) {
+        it.currentUrl != URLs.APP_URL_HOME
```

Comment:
Nit: we have an extension class that makes this a little more readable.  Should look like
```kotlin
sessionRepo.state.map {
    it.currentUrl != URLs.APP_URL_HOME
}
```

Reply:
Oh nice, of course that would have been written because it's so useful :P